### PR TITLE
Create card.css

### PR DESCRIPTION
**Pull request by pnchinmay**

### CODE REVIEW

**Comment by Chiragmodi01** [comment]
That looks awasome!

File: card.css [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (Chiragmodi01) [inline card.css:1]: Change it to 'hello world'!

**Review by Chiragmodi01 — CHANGES_REQUESTED** [review]
Looks good!

File: card.css (hunk 1 above)
Comment (pnchinmay) [reply card.css:1]: will add another commit

**Review by Chiragmodi01 — APPROVED** [review]

**Pull request merged**